app/src/main/java/com/github/mik629/android/fundamentals/ui/movieslist/MoviesListViewModel.kt

```diff
@@ -0,0 +1,42 @@
+package com.github.mik629.android.fundamentals.ui.movieslist
+
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
+import com.github.mik629.android.fundamentals.domain.model.Movie
+import com.github.mik629.android.fundamentals.domain.repositories.MoviesRepository
+import kotlinx.coroutines.launch
+import timber.log.Timber
+
+class MoviesListViewModel(
+    private val moviesRepository: MoviesRepository
+) : ViewModel() {
+
+    private val cached = HashMap<Long, Movie>()
```

Comment:
lets create 2 separate viewModels

Reply:
yeah, already created, sorry, this `cache` is not used any more, forgot to remove